src/main/java/org/jenkins/plugins/lockableresources/queue/LockRunListener.java

```diff
@@ -28,20 +26,20 @@
 import org.jenkins.plugins.lockableresources.actions.ResourceVariableNameAction;
 
 @Extension
-public class LockRunListener extends RunListener<AbstractBuild<?, ?>> {
+public class LockRunListener extends RunListener<Run<?, ?>> {
 
 	static final String LOG_PREFIX = "[lockable-resources]";
 	static final Logger LOGGER = Logger.getLogger(LockRunListener.class
 			.getName());
 
 	@Override
-	public void onStarted(AbstractBuild<?, ?> build, TaskListener listener) {
+	public void onStarted(Run<?, ?> build, TaskListener listener) {
```

Comment:
Note that I kept the signature as `Run` for consistency with the `onDeleted` and `onCompleted` methods - I'd rather the consistent signature and handling of the `AbstractBuild` case than inconsistent signatures. But I'm flexible. =)